.github/workflows/release.yml

```diff
@@ -185,7 +185,7 @@ jobs:
           JIRA_DESCRIPTION: ${{ env.jira_description }}
```

Comment:
1. It looks like the code changes are modifying the format of the RELEASE_NOTES variable by removing the extra line breaks and formatting characters. 

2. The sed commands are being used to manipulate the string, but it would be beneficial to add comments explaining what each sed command is doing for better understanding.

3. Consider breaking down the sed commands into separate lines or variables for better readability and maintainability.

4. Make sure to test the changes thoroughly to ensure that the new format of the RELEASE_NOTES variable is correct and doesn't break any functionality.